mongo/client_bulk_write.go

```diff
@@ -50,11 +50,11 @@ type clientBulkWrite struct {
 
 func (bw *clientBulkWrite) execute(ctx context.Context) error {
 	if len(bw.writePairs) == 0 {
-		return ErrEmptySlice
+		return fmt.Errorf("error from writes: %w", ErrEmptySlice)
```

Comment:
Optional: Consider a more descriptive error message.

E.g.
```go
fmt.Errorf("empty writes slice: %w", ErrEmptySlice)
```

Reply:
I think "empty writes slice" is redundant because the message in `ErrEmptySlice` has stated the slice should be non-empty.

Reply:
My goal is to clarify which `BulkWrite` parameter caused the validation error. "error from writes" is somewhat ambiguous because it could be interpreted as meaning there was an error performing the writes on the server. Would a less redundant error like "invalid writes" be better?